PowerEditor/installer/nativeLang/italian.xml

```diff
@@ -18,9 +18,9 @@
                     <Item menuId="language" name="&amp;Linguaggio"/>
                     <Item menuId="settings" name="C&amp;onfigurazione"/>
                     <Item menuId="tools" name="&amp;Strumenti"/>
-                    <Item menuId="macro" name="Macro"/>
-                    <Item menuId="run" name="Esegui"/>
-                    <Item idName="Plugins" name="Plugin"/>
+                    <Item menuId="macro" name="M&amp;acro"/>
+                    <Item menuId="run" name="Ese&amp;gui"/>
+                    <Item idName="Plugins" name="P&amp;lugin"/>
```

Comment:
Apparently, the letter `l` is used twice as a menu keyboard accelerator — in `P&amp;lugin` and `&amp;Linguaggio`.
Perhaps, `&amp;Plugin`?